perf/run-benchmarks

```diff
@@ -271,6 +272,8 @@ def run_semgrep(
         args.extend(["--time"])
     if trace:
         args.extend(["--trace"])
+    if traces_endpoint:
+        args.extend(["--traces-endpoint", traces_endpoint])
```

Comment:
omg thanks for also modifying the benchmarks! One thing: it might actually be nice to make the default for benchmarks `semgrep-dev` when `--trace` is set but `--traces-endpoint` isn't